### DISCUSSION TITLE
Do we want a dmg for the macOS client?

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#4995] [Ideas]
Playing around with some software and ideas, and wondered if we wanted a dmg for the macOS client?

![SCR-20230223-3l5](https://user-images.githubusercontent.com/69029666/220635795-7b44ce34-ce55-4349-9005-c714eb05df41.png)

@coeur @sweetppro

**Comment by Coeur** [comment]
Duplicate of #4062 ? :)

**Comment by ckerr** [comment]
Seems a little silly to convert to an issue so that I can close it as a dupe :smile_cat: 

I'll just lock this one instead
